EventsExpress/ClientApp/src/components/event/event-item-view.js

```diff
@@ -70,23 +70,51 @@ export default class EventItemView extends Component {
         ));
     }
 
+
+    getClassName = (attitude) => {
+        switch (attitude) {
+            case 0:
+                return "attitude-like";
+            case 1:
+                return "attitude-dislike";
+            default:
+                return '';
+        }
+    }
```

Comment:
Let move this into a separate function and import every time is needed to avoid duplication